modules/tinymce/src/core/test/ts/browser/keyboard/EnterKeyListsTest.ts

```diff
@@ -509,4 +510,56 @@ describe('browser.tinymce.core.keyboard.EnterKeyListsTest', () => {
     assert.equal(editor.getContent(), '<ul><li>te<br />xt</li></ul>');
     editor.settings.forced_root_block = 'p';
   });
+
+  it('TINY-5974: Should be able to outdent empty list using enter key', () => {
+    const editor = hook.editor();
+    editor.setContent(
+      '<ul>' +
+        '<li>' +
+          'a' +
+          '<ul>' +
+            '<li>\u00a0</li>' +
+          '</ul>' +
+        '</li>' +
+      '</ul>'
+    );
+    editor.focus();
```

Comment:
Why are we calling `focus` for each test here, as that shouldn't be necessary?

Reply:
Ahh, many others do that in this file. I thought we had cleaned that up in the conversion, but seems some got missed. We probably should remove them all and just tell `TinyHooks` to focus on init. If it is needed though, we probably should have that in a `beforeEach`.

Anyways, largely out of scope so we can clean that up independently.

Reply:
That was correct, `focus` wasn't needed for this test so I removed that here.